posthog/api/test/test_feature_flag.py

```diff
@@ -738,9 +941,50 @@ def test_validation_group_properties(self):
             },
         )
 
-    def _create_flag_with_properties(self, name, properties, **kwargs):
-        return self.client.post(
-            f"/api/projects/{self.team.id}/feature_flags/",
+    def _create_flag_with_properties(
+        self,
+        name: str,
+        properties,
+        team_id: Optional[int] = None,
+        expected_status: int = status.HTTP_201_CREATED,
+        **kwargs,
+    ):
+        if team_id is None:
+            team_id = self.team.id
+
+        create_response = self.client.post(
+            f"/api/projects/{team_id}/feature_flags/",
             data={"name": name, "key": name, "filters": {**kwargs, "groups": [{"properties": properties,}],},},
             format="json",
         )
+        self.assertEqual(create_response.status_code, expected_status)
+        return create_response
+
+    def _get_feature_flag_activity(
+        self, flag_id: int, team_id: Optional[int] = None, expected_status: int = status.HTTP_200_OK
+    ):
+        if team_id is None:
+            team_id = self.team.id
+
+        activity = self.client.get(f"/api/projects/{team_id}/feature_flags/{flag_id}/activity")
+        self.assertEqual(activity.status_code, expected_status)
+        return activity.json()
+
+    def assert_feature_flag_activity(self, flag_id: int, expected: List[Dict]):
+        """
+        should have 5 queries
+        select django_session
+        select user
+        select team
+        select organization
+        check flag exists
+        select historical versions (with user details)
+        """
+        with self.assertNumQueries(6):
```

Comment:
rather than checking queries explicitly can we:

1. create feature flag as user 1
2. request endpoint and get query count
3. update feature flag as user 2
4. request endpoint and verify counts are the same

this way we're just testing how it scales, not the implementation details of the endpoint